### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -L filetests,inout,nclude`

### CODE REVIEW

File: building/markdown/markdown.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ We support special types of blocks that can be added to documents to pull out co
 We support three types of blocks:
 
 - **exercism/note:** Blocks that pull out some extra special information
-- **exercism/caution:** Things that people should know about or tread carefully with
+- **exercism/caution:** Things that people should know about or treat carefully with
```
Comment (ErikSchierboom) [inline building/markdown/markdown.md]: I think this change is not correct. One treads carefully, not treats carefully

**Review by ErikSchierboom — COMMENTED** [review]
Thanks! One tiny nit

**Review by ee7 — APPROVED** [review]
Approved, conditional on making the change that Erik suggested.

**Review by ErikSchierboom — APPROVED** [review]
Thanks!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**